languages/csharp/Bitwarden.Sdk/Bitwarden.Sdk.csproj

```diff
@@ -1,11 +1,11 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net6.0</TargetFramework>
```

Comment:
Question: Does increasing this affect people using the SDK in older .NET versions?

Reply:
Yes :/ .NET 6 does go out of support in November though. My current thinking is that I will try and convert more of the commands to be Async and then maybe we do a `1.0.0` release after .NET 6 goes out of support with the breaking changes of converting them.

Reply:
Do we need to upgrade target framework? We've had requests to use NET Standard 2.0 to support .NET Framework.

Reply:
`[LibraryImport]` requires .NET 7+, in my opinion supporting .NET Framework would not be worth it. C# may soon be getting built in `Result` and `Option` types that we could take advantage of here and getting locked into .NET Framework support would either mean a of conditional compilation or just holding back the .NET targets.